src/test/java/com/stripe/BaseStripeTest.java

```diff
@@ -113,7 +113,7 @@
 				Mockito.<Map<String, Object>>any(),
 				Mockito.<Class<T>>any(),
 				Mockito.any(APIResource.RequestType.class),
-				Mockito.any(RequestOptions.class))).thenReturn(APIResource.GSON.fromJson(response, clazz));
+				Mockito.<RequestOptions>any())).thenReturn(APIResource.GSON.fromJson(response, clazz));
```

Comment:
Had to change this because in Mockito 2, the `any(SomeClass.class)` no longer matches `null`.

Reply:
Makes sense. Thanks.